.github/workflows/lint-and-test.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on:
+  push:
+    
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: dependencies
+      run: yarn install
```

Comment:
Can we have cache? 😁

Reply:
I guess this works cfbf96b 😅.

Also, @julien-leclercq, does adding the condition `if: steps.cache-primes.outputs.cache-hit != 'true'` will produce the issue you were talking this morning ? i.e having to add, as a first job
```yaml
success:
    name: "success"
    runs-on: ubuntu-latest
    steps:
      - name: "success"
        run: echo "success"
```

Would be great if you could explain real quick the deal to @Fenntasy.